code/game/machinery/vendor/medical_vendors.dm

```diff
@@ -72,12 +73,13 @@
 
 		/obj/item/stack/medical/bruise_pack/handmade = 25,
 		/obj/item/stack/medical/bruise_pack = 50,
-		/obj/item/stack/medical/bruise_pack/soteria = 700,
+		/obj/item/stack/medical/bruise_pack/soteria = 250,
 		/obj/item/stack/medical/ointment/handmade  = 25,
 		/obj/item/stack/medical/ointment = 35,
-		/obj/item/stack/medical/ointment/soteria = 700,
+		/obj/item/stack/medical/ointment/soteria = 250,
 		/obj/item/stack/medical/splint/improvised = 5,
 		/obj/item/stack/medical/splint = 10,
+		/obj/item/storage/firstaid/regular/si = 750,
```

Comment:
This has about 1310cr value (1250 for bruise_pack and ointment + 50 + 10)
Maybe lower the price of the SI stuff a little more or remove the spare bandaids so its not as much of a swing in prices

Reply:
Alright. Will do.